evap/grades/forms.py

```diff
@@ -0,0 +1,45 @@
+from django import forms
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+from evap.evaluation.forms import BootstrapMixin
+
+from evap.grades.models import GradeDocument
+
+
+class StrippedCharField(forms.CharField):
+    """
+        CharField that saves trimmed strings
+    """
+
+    def to_python(self, value):
+        super(StrippedCharField, self).to_python(value)
+        if value is None:
+            return None
+        return value.strip()
+
+
+class GradeDocumentForm(forms.ModelForm, BootstrapMixin):
+    description = StrippedCharField(max_length=255)
+    last_modified_time_2 = forms.DateTimeField(label=_("Last modified"), required=False, localize=True)
+    last_modified_user_2 = forms.CharField(label=_("Last modified by"), required=False)
```

Comment:
ok. a comment in one place and references to that comment in the other places would be cool.